src/app.rs

```diff
@@ -155,8 +155,10 @@ impl<'a, 'b> Application<'a, 'b> {
         {
             let mut time = self.engine.world.write_resource::<Time>();
             time.frame_number += 1;
+            time.delta_seconds = duration_to_secs(self.time.delta_time);
```

Comment:
I can't say I like that `time.fixed_step` and `time.fixed_seconds` can differ due to human error. Maybe we should put this behind a method to set/get the fields?